polygeist/include/mlir/Dialect/Polygeist/Utils/TransformUtils.h

```diff
@@ -230,6 +237,66 @@ class LoopTools {
                    const LoopVersionCondition &versionCond) const;
 };
 
+//===----------------------------------------------------------------------===//
+// VersionConditionBuilder
+//===----------------------------------------------------------------------===//
+
+using AccessorType = TypedValue<MemRefType>;
```

Comment:
I would place the using declaration inside the `VersionConditionBuilder` class to avoid polluting the mlir namespace (given that now this class is in a header file).

Reply:
done.